README.md

```diff
@@ -77,6 +77,36 @@ than what many distros are currently shipping. This should become less of an
 issue in the future. For the time being, the following custom repositories
 are provided for select distros.
 
+### Ubuntu
+
+Ubuntu is supported through the following launchpad project.
```

Comment:
Just to make it explicitly clear that this is not yet officially supported by Ubuntu, maybe we should rephrase this as:

Experimental sched_ext support for Ubuntu is provided by the following launchpad project:

Sentence should end in a colon either way